Development_Rules.md

```diff
@@ -0,0 +1,21 @@
+# Development flow
+
+* For every feature/bug fix - new branch needs to be created
+    * feature - feature/
+    * bug fix / fixes - fix
+* That feature/bug fix branch should also contain tests
+* Merge request is only created when work with branch is done 
+    * if you want to check if there is some kind of chuck comments - just create a merge request with `WIP:` at the start
+* Branch needs to be removed after merge request is merged
+
```

Comment:
I think it would be nice if we'd also add task number in brach name

Reply:
Good point. Or we could also add Jira Task link in the description instead. @Dezash , @lukasrakauskas what do you think?

Reply:
I think it might be worth looking into integrating Jira with GitHub so that we can perform actions using commands in our commit messages.
https://confluence.atlassian.com/adminjiracloud/connect-jira-cloud-to-github-814188429.html